diff -aurN Terminal-0.4.0.orig/terminal/terminal-preferences.c Terminal-0.4.0/terminal/terminal-preferences.c
--- Terminal-0.4.0.orig/terminal/terminal-preferences.c	2009-07-20 20:03:50.000000000 +0200
+++ Terminal-0.4.0/terminal/terminal-preferences.c	2009-08-13 21:30:39.000000000 +0200
@@ -990,7 +990,7 @@
                                    g_param_spec_string ("font-name",
                                                         "font-name",
                                                         "FontName",
-                                                        "Monospace 12",
+                                                        "Monospace 10",
                                                         EXO_PARAM_READWRITE));
 
   /**
@@ -1078,7 +1078,7 @@
                                    g_param_spec_boolean ("misc-menubar-default",
                                                          "misc-menubar-default",
                                                          "MiscMenubarDefault",
-                                                         TRUE,
+                                                         FALSE,
                                                          EXO_PARAM_READWRITE));
 
   /**
@@ -1179,7 +1179,7 @@
                                                       "scrolling-bar",
                                                       "ScrollingBar",
                                                       TERMINAL_TYPE_SCROLLBAR,
-                                                      TERMINAL_SCROLLBAR_RIGHT,
+                                                      TERMINAL_SCROLLBAR_NONE,
                                                       EXO_PARAM_READWRITE));
 
   /**
